contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -242,7 +244,8 @@ var ChannelModel = BaseModel.extend({
 		author: "Anonymous",
 		license_owner: "No license found",
 		description:" ",
-		thumbnail_url: "/static/img/kolibri_placeholder.png"
+		thumbnail_url: "/static/img/kolibri_placeholder.png",
```

Comment:
Defaults so that channel page can function without saving to the db automatically